README.md

```diff
@@ -1,32 +1,60 @@
 # Setup
 
-install docker. This is the hard part.
-
-## prerequisites
-
-- Virtualbox - this could probably change
-- helpful script (drop in your bashrc). sets env and echos the ip of the new container
+## Prerequisites
+
+zshell is recommended. The fish shell has been problematic.
+
+1. `cd app; npm install`
+1. Install Virtualbox (tested on 5.0.6)
+1. Install Docker (this is the hard part)
+    1. `brew install docker` OR download standalone installer
+    1. `sudo chown -R [youruser]:staff ~/.docker`
+    1. `docker-machine create --driver virtualbox dev`
+    1. `eval $(docker-machine env dev)` // Why is this run?
```

Comment:
`docker-machine env default` is a command whose output is a script which, when run, sets the correct variables in the environment.

`eval $(SOMETHING)` will evaluate the expression `SOMETHING` in the context of the current shell.

combining the two you have a script generated which is then evaluated (executed) within the context of the current shell. End result being that you have the correct variables set in your current shell session.